src/test/java/de/mediathekview/mserver/crawler/CrawlerManagerFTPTest.java

```diff
@@ -42,10 +42,8 @@
   private static final String FILMLISTS_TARGET_BASE_PATH = "/var/www/mediathekview/filmlisten/";
   private static final String FTP_PASSWORD = "!fTpSeCuReD";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') 'PASSWORD' detected in this expression, review this potentially hardcoded credential. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)